searchlib/src/vespa/searchlib/fef/indexproperties.cpp

```diff
@@ -49,10 +50,12 @@ uint32_t
 lookupUint32(const Properties &props, const vespalib::string &name, uint32_t defaultValue)
 {
     Property p = props.lookup(name);
+    uint32_t value(defaultValue);
     if (p.found()) {
-        return atoi(p.get().c_str());
+        const auto & s = p.get();
+        std::from_chars(s.c_str(), s.c_str() + s.size(), value);
```

Comment:
Is there a chance that these properties may have leading whitespace? `atoi` implicitly consumes these while `from_chars` does not.

Reply:
I expect no leading whitespace, but I will make it atoi compliant anyway to reduce risk of regression.

Reply:
Fixed and tested explicit